src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -86,7 +86,15 @@
     static {
         Properties prop = new Properties();
         try {
-            prop.load(new FileInputStream("src/test/resources/test.properties"));
+            String propertiesFilename;
+            if (isTravis()) {
+                propertiesFilename = "test.travis.properties";
+            } else if (isAppveyor()) {
+                propertiesFilename = "test.appveyor.properties";
+            } else {
+                propertiesFilename = "test.properties";
+            }
+            prop.load(new FileInputStream("src/test/resources/" + propertiesFilename));
```

Comment:
Again, I don't feel like we need to go this far (having separate `.properties` files) just for this. Is it possible to just do everything via `TestProperties.java`?
```java
...
// set all properties as before
if (isTravis()) {
    ...
}
if (isAppveyor()) {
    ...
}
...
```